assets/js/spec/game_test.js

```diff
@@ -7,8 +9,9 @@ function makeResponse(object) {
     }
 }
 
-function makeStartResponse(playerId) {
-    return makeResponse({'player-id':playerId});
+function makeStartResponse(playerId, gameId, mark) {
+    return makeResponse({
+        'player-id':playerId, 'game-id': gameId, 'mark': mark});
```

Comment:
If objects get big enough that they have to be broken I think it reads better to have each property on a new line like https://google.github.io/styleguide/javascriptguide.xml#Code_formatting